community/mit-scheme/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+cd src
+./configure
```

Comment:
```sh
./configure \
    --prefix=/usr
```

Reply:
Hey Dylan thanks for your review. See the latest commit 4c943d39ec145208043885d328f58a57b989505f for the changes. I also updated the manifest